cortexlab/+git/reset.m

```diff
@@ -0,0 +1,26 @@
+function status = reset(file)
```

Comment:
what is the point of having this function?

Reply:
Originally I had changed the paths file itself for the tests.  Now I'm using the custom paths file.  It may be useful in the future, however we could remove it for now as it's not used anywhere

Reply:
By the way see issue #172 and use this file as an example of how to resolve it

Reply:
ok, I'll remove it, but keep it in mind.